python/ray/ml/train/integrations/horovod/horovod_trainer.py

```diff
@@ -0,0 +1,8 @@
+from ray.ml.train.data_parallel_trainer import DataParallelTrainer
+
+from ray.train.horovod import HorovodConfig
+
+
+class HorovodTrainer(DataParallelTrainer):
+    def __init__(self, train_loop_per_worker, backend_config: HorovodConfig, **kwargs):
```

Comment:
Can you add docstring for this? You can copy whats in TorchTrainer or TensorflowTrainer.